js/gluon.js

```diff
@@ -1,7 +1,7 @@
 $(function() {
     // constants
-    var MIN_JDK = "21";
-    var JDK_MAJOR = "23";
+    var MIN_JDK = "22";
```

Comment:
It is a bit trickier since the current OpenJFX LTS versions (21 and 17) have 21 as the minimal JDK version.